ibis/backends/sqlite/tests/test_types.py

```diff
@@ -0,0 +1,76 @@
+import datetime
+import sqlite3
+
+import pandas as pd
+import pytest
+
+import ibis
+import ibis.expr.datatypes as dt
+
+
+def to_datetime(value):
```

Comment:
It seems fine to use the function from `ibis.backends.sqlite` instead of duplicating it here.

Reply:
Done.